packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.js

```diff
@@ -140,48 +142,24 @@ const GiftCards = props => {
             >
                 {`Apply`}
             </Button>
-            {canTogglePromptState && (
-                <Trigger
-                    action={togglePromptState}
-                    classes={{ root: classes.toggle_button }}
-                >
-                    <Icon src={CloseIcon} />
-                </Trigger>
-            )}
-            {shouldDisplayCardBalance && (
-                <div className={classes.balance}>
-                    <span>{checkBalanceData.code}</span>
-                    <span className={classes.price}>
-                        {`Balance: `}
-                        <Price
-                            value={checkBalanceData.balance.value}
-                            currencyCode={checkBalanceData.balance.currency}
-                        />
-                    </span>
-                </div>
-            )}
             <button
                 className={classes.check_balance_button}
                 disabled={isCheckingBalance}
                 onClick={checkGiftCardBalance}
             >
-                {`Check Gift Card Balance`}
+                {`Check balance`}
```

Comment:
Haven't seen template literals without interpolation in the codebase before. Not requesting a change, but just pointing out it seems inconsistent; we may want to consider aligning on this and adding a lint rule. Speaking of adding lint rules, @jimbo discussed a list of rules he anticipated on adding, it would be good to share so we can add and discuss anything new that comes up.

Reply:
Throw it on the punt pile!

Reply:
Done in b87b2e757